### ISSUE TITLE
CO2 is not an available molecule

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/molecule-shapes-basics#12]
**Test device:**
Dell
**Operating System:**
Win 10
**Browser:**
chrome
**Problem description:**
I noticed that the menu picture for the Real Molecules screen is a CO2 molecule, but CO2 is not one of the molecules you can look at in the Basics sim.
**Steps to reproduce:**
1.  Look at menu
2. Look at second screen molecule options

**Screenshots:**
![co2menu](https://user-images.githubusercontent.com/41024075/56840886-6223f080-6847-11e9-88f8-33f313f2cbf9.png)
![noco2](https://user-images.githubusercontent.com/41024075/56840887-6223f080-6847-11e9-97cf-f39383d10923.png)

Troubleshooting information (do not edit):

<details>

Name: ‪Molecule Shapes: Basics‬
URL: https://phet.colorado.edu/sims/html/molecule-shapes-basics/latest/molecule-shapes-basics_en.html
Version: 1.2.1 2019-04-25 00:33:59 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/74.0.3729.108 Safari/537.36
Language: en-US
Window: 1536x722
Pixel Ratio: 2.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by emily-phet** [comment]
@ariel-phet 
I suggested @KatieWoe make an issue for this, after she pointed out the discrepancy to me. I have no idea if any work is currently occurring or planned for this sim. No urgency from me, just wanting to have this noted for the next time there is work on the Mol Shapes pair of sims.

**Comment by ariel-phet** [comment]
@emily-phet @KatieWoe thanks for pointing out. No work for this sim planned currently, will mark as deferred

**Comment by KatieWoe** [comment]
Yup, sorry I didn't think of it like a month ago when it was being re-released.

**Comment by arouinfar** [comment]
The 1.7 version is in the works over in https://github.com/phetsims/molecule-shapes/issues/263.

I can create a new screen icon.

**Comment by arouinfar** [comment]
The screen icons are code-generated, but tweaking them was straightforward. Addressed in the above commit. Summary of changes:
* Swap out CO2 for the linear molecule actually available in the Basics version, BeCl2.
* Use single bonds in the Model icon to match the bond order of BeCl2.

Before:
<img width="807" height="352" alt="Image" src="https://github.com/user-attachments/assets/1821f729-d940-4773-91b0-6bc4e66841b5" />

After:
<img width="802" height="342" alt="Image" src="https://github.com/user-attachments/assets/eda08247-eed5-4b10-adec-d027e6a1698d" />
